mockserver/__init__.py

```diff
@@ -64,6 +64,10 @@ def form(form):
     return collections.OrderedDict((("type", "PARAMETERS"), ("parameters", _to_named_values_list(form))))
 
 
+def request_json(payload, match_type="STRICT"):
```

Comment:
i think the contract would be more intuitive if this was instead named "strict_json"

that way the code would look like `request(body=strict_json(....))` which i think is a little more fluent since we already know we're in a request.  (i would've even said body=json()  but that would conflict with the json package

As for the match type, since mockserver only support 2 options for json, and they could be 2 different body type (it's the only body type with a behvior switching param after all).   The goal of this library is to remain simple and straight forward for the user and not the writer.  So we could add `request(body=json_fields(....))` right beside it to cover both cases.   (i'm open to another name   `json_with_fields` ?)

Reply:
> (i would've even said body=json() but that would conflict with the json package

that was my exact thinking :-)

> As for the match type, since mockserver only support 2 options for json, and they could be 2 different body type (it's > the only body type with a behvior switching param after all). The goal of this library is to remain simple and straight > forward for the user and not the writer. So we could add request(body=json_fields(....)) right beside it to cover both > cases. (i'm open to another name json_with_fields ?)
I wanted to be flexible at this point for possible changes in mock-server. But since it's just two options like you said I'll just change it for this case. Personally I like `json_with_fields` more - it's a bit clearer.